thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java

```diff
@@ -92,9 +94,63 @@ public Long cloneFunctionGetId(@PathParam("id") Long id,
     }
   }
 
+  /**
+   * Copy the Identity of source anomaly function (with srcId) over in total to destination anomaly function (with destId)
+   * Explicit representation: denote source anomaly function with (srcId, srcFunctionName, srcProperties)
+   *                          denote destination anomaly function with (destId, destFunctionName, destProperties)
+   * "copyIdentity" will update destination anomaly function into source anomaly function's identity
+   * After "copyIdentity", the two functions will become:
+   *        (srcId, srcFunctionName, destProperties)
+   *        (destId, destFunctionName, destProperties)
+   * This in fact updates source anomaly function's properties into destination function's properties
+   * @param srcId : the source function with Identity to be copied to
+   * @param destId : the destination function Id that have properties to be copied to source function
+   * @return OK is success
+   */
+  @POST
+  @Path("function/{srcId}/copyIdentity/{destId}")
```

Comment:
Refer this to @bl44 or @puneetjaiswal. Please make sure the path is consistent with all the others.